source/analyze-code.txt

```diff
@@ -121,6 +121,28 @@ information message displayed:
       .. figure:: /includes/images/builder-popup-photoshop.png
          :alt: Screenshot of builder expression in visual studio with information message displayed.
 
+The {+product+} supports builders variables tracking and composition. You can
+combine multiple builders expressions with `logical operators
```

Comment:
It looks like the rest of the page refers to these as "builder expresssions" (singular builder)

```suggestion
combine multiple builder expressions with `logical operators
```